boards/arm/mps2_an521/mps2_an521.yaml

```diff
@@ -9,3 +9,5 @@ toolchain:
   - xtools
 supported:
   - gpio
+testing:
+  default: true
```

Comment:
if this is going to be used to test the architecture, I would exclude many non-kernel/non-arch tests here using tags. Otherwise this will just make CI slower for no good reason (for example, disable all bluetooth, networking and other areas not relevant to what this board is testing)

Reply:
Makes sense. Ideally I would like all -arm -userspace and -kernel tests build & execute; the rest I don't care that much

Reply:
@nashif i thought we have one platform per ARCHitecture doing everything, though....